crm_compassion/models/crm_lead.py

```diff
@@ -54,6 +55,16 @@ def create_event(self):
             "context": context,
         }
 
+    @api.multi
+    @api.depends("event_ids")
+    def _compute_planned_sponsorship(self):
+        for lead in self:
+            future_planned_sponsorships=0
+            for e in lead.event_ids:
+                if e.start_date > datetime.datetime.now():
+                    future_planned_sponsorships+=e.planned_sponsorships
+            lead.planned_sponsorships=future_planned_sponsorships
```

Comment:
This is just flake8 formatting issues, that requires putting spaces around the operators.
```suggestion
            future_planned_sponsorships = 0
            for e in lead.event_ids:
                if e.start_date > datetime.datetime.now():
                    future_planned_sponsorships += e.planned_sponsorships
            lead.planned_sponsorships = future_planned_sponsorships
```